css/layout.css

```diff
@@ -14,6 +14,12 @@ body {
   padding: 20px;
 }
 
+.row4 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
```

Comment:
When we were talking through the styling together I mentioned this but in the future we won't want to have `row1`, `row2`, `row3`, `row4`. . . there should just be one `row` that is `display:flex` and then other classes for your justify-content and align-items.